build.gradle

```
@@ -35,6 +35,7 @@ dependencies {
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 
 	runtimeOnly 'com.h2database:h2'
+	runtimeOnly "mysql:mysql-connector-java"
```

```suggestion
	runtimeOnly 'mysql:mysql-connector-java'
```

소소하지만..! 컨벤션을 맞추는게 좋아보이네요 ㅎㅎ